### PR TITLE
Refactor torque stuff

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ repos:
     rev: v0.931
     hooks:
     -   id: mypy
-        exclude: '^(pyextra/)|(cereal/)|(rednose/)|(panda/)|(laika/)|(opendbc/)|(laika_repo/)|(rednose_repo/)/'
+        exclude: '^(pyextra/)|(cereal/)|(rednose/)|(panda/)|(laika/)|(opendbc/)|(laika_repo/)|(selfdrive/car/torque_data/)|(rednose_repo/)/'
```
Comment (adeebshihadeh) [inline .pre-commit-config.yaml:22]: why is this necessary?

File: selfdrive/car/torque_data/override.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+FORD FOCUS 4TH GEN: [.nan, 1.5, .nan]
+###
+
+# No steering wheel
+COMMA BODY: [.nan, 1000, .nan]
+
+# Totally new car
+KIA EV6 2022: [3.0, 2.5, 0.05]
+
+# Ideal car
```
Comment (adeebshihadeh) [inline selfdrive/car/torque_data/override.yaml:25]: dashcam or fallback car is more accurate

File: selfdrive/car/interfaces.py [hunk 3]
```diff
@@ -109,9 +111,22 @@ def get_std_params(candidate, fingerprint):
 
   @staticmethod
   def get_torque_params(candidate, default=float('NaN')):
+    with open(TORQUE_SUBSTITUTE_PATH) as f:
```
Comment (adeebshihadeh) [inline selfdrive/car/interfaces.py:114]: this should also enforce that each car is only defined once

File: selfdrive/car/torque_data/override.yaml (hunk 2 above)
Comment (HaraldSchafer) [reply selfdrive/car/torque_data/override.yaml:25]: for the KIA?

File: selfdrive/car/interfaces.py (hunk 3 above)
Comment (HaraldSchafer) [reply selfdrive/car/interfaces.py:114]: willdo

File: selfdrive/car/torque_data/override.yaml (hunk 2 above)
Comment (adeebshihadeh) [reply selfdrive/car/torque_data/override.yaml:25]: for mock

Comment (HaraldSchafer) [reply selfdrive/car/torque_data/override.yaml:25]: Oh I see
